### PR TITLE
remove one ignore clause for module copy

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Remove the cases in `ignore.txt` for the module `copy`.

Case 1: doc-default-does-not-match-spec
Adjust doc and code to match.

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request

### CODE REVIEW

File: changelogs/fragments/83643-fix-sanity-ignore-for-copy.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - copy - parameter ``local_follow`` was incorrectly documented as having default value ``True`` (https://github.com/ansible/ansible/pull/83643).
+  - copy - minor refactor (https://github.com/ansible/ansible/pull/83643).
```
Comment (flowerysong) [inline changelogs/fragments/83643-fix-sanity-ignore-for-copy.yml]: This changelog entry is a little confusing. I don't see how this PR could be seen as a refactor? Adding `default=False` doesn't change anything about the structure of the code, it's just a cosmetic change to make the test happy.

Comment (russoz) [reply changelogs/fragments/83643-fix-sanity-ignore-for-copy.yml]: Totally agree and happily accepting suggestions. I was/am not sure how to describe something that will have no impact in the users' life.

Comment (flowerysong) [reply changelogs/fragments/83643-fix-sanity-ignore-for-copy.yml]:
```suggestion
  - copy - fix sanity test failures (https://github.com/ansible/ansible/pull/83643).
```

**Comment by russoz** [comment]
hi @Akasurde m could you please confirm if there is anything else needed in this PR? TIA

**Comment by Akasurde** [comment]
> hi @Akasurde m could you please confirm if there is anything else needed in this PR? TIA

Sure I will check.

**Review by Akasurde — APPROVED** [review]

**Pull request merged**